2-mandatory/2-water-bottle.js

```diff
@@ -19,18 +19,31 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    return (this.volume = 100);
   },
-  pour: function () {
+  pour: function (units) {
     // calling this function should increase your bottle volume by 10 units;
+ if (this.volume === 0) {
+   return (this.volume += 10);
```

Comment:
I think it is doing an update operation, so it is not necessary to use the return keyword.